### ISSUE TITLE
aria-roledescription flagged as an issue for role="slider"

### ISSUE THREAD

**Issue by pavandixit93** [dequelabs/axe-core#2598]
I ran axe on input with type=range and `role="slider"` with the `aria-roledescription` attribute and axe flagged it as a critical issue. Here is the issue description copied and pasted from axe:

> Use aria-roledescription on elements with a semantic role
> Issue description
> Ensure aria-roledescription is only used on elements with an implicit or explicit role
> 
> Impact: critical
> 
> 
> This potential issue needs your review...
> Check that the aria-roledescription is announced by supported screen readers
> But aria-roledescription is allowed in the spec: [https://www.w3.org/TR/wai-aria-1.1/#slider](https://www.w3.org/TR/wai-aria-1.1/#slider), in `Inherited States and Properties` column we can see the same.
> 
FYI: Screen readers does announce the `aria-roledescription` set by user.
Example link: [https://stackblitz.com/edit/aria-roledescription-and-role-slider-issue](https://stackblitz.com/edit/aria-roledescription-and-role-slider-issue)
Axe shouldn't flag this as an issue per the spec.

This is critical issue, please give me some suggestions/documentation.

**Comment by straker** [comment]
Thanks for the issue. [We did some extensive testing of aria-roledescription](https://codepen.io/straker/full/Xoybem) last year, which determined how we handle our current rule. Specifically when we tested `<input type=range>` both JAWS and Talkback did not support it (didn't announce the aria-roledescription value). 

Doing the tests again for the input:

```html
<label for="input5">Implicit</label>
<input type="range" id="input5" name="input5" aria-roledescription="custom role"/>

<label for="input5">Explicit </label>
<input type="range" role="slider" id="input5" name="input5" aria-roledescription="custom role"/>
```

Talkback and JAWS 2020.2008.24 in IE11 and Edge (but Chrome works) still fail to announce the aria-roledescription value. So I would say this is still a valid issue and should be flagged as needs review (which looks like it does).

**Comment by pavandixit93** [comment]
@straker 
Thank you for quick response :), I had tested on Chrome only. This seems valid issue as JAWS doesn't announce text in IE11 & Edge. 
Closing for now.
